src/graph_gui.cpp

```diff
@@ -200,13 +206,17 @@ struct BaseGraphWindow : Window {
 
 	StringID format_str_y_axis;
 	uint8_t colours[GRAPH_MAX_DATASETS];
+
```

Comment:
There are a lot of empty lines being added and removed in this PR, not touching any of your changes. Please take a look and leave them unchanged to keep the diff size down, unless they actually need to be changed. 🙂

Reply:
Still relevant: Don't add newline. 😉